indico/MaKaC/webinterface/tpls/CategModifMain.tpl

```diff
@@ -1,3 +1,5 @@
+${ render_template('flashed_messages.html') }
```

Comment:
Do we have ugly full-page-width flashes on that page? If yes maybe we could use `render_template('flashed_messages.html', fixed_width=true)` and add change flashed_messages.html like this: ` {{ message_box(category, message, fixed_width=fixed_width|default(false)) }}` (the default is not really needed, but I think passing false instead of Undefined is cleaner)